churaverse-plugins-client/src/synchroBreakPlugin/synchroBreakPlugin.ts

```diff
@@ -323,8 +325,8 @@ export class SynchroBreakPlugin extends CoreGamePlugin {
   /**
    * ゲームの説明ウィンドウを取得する
    */
-  private getDescriptionWindow(): IDescriptionWindow {
-    const descriptionWindow = this.gamePluginStore.gameUiManager.getUi(this.gameId, 'descriptionWindow')
+  private get getDescriptionWindow(): IDescriptionText {
```

Comment:
getterは関数でありながら変数のような見た目でアクセスできる、なのでgetterを使うのであれば命名も変数に寄せた方が良いです
```suggestion
  private get descriptionWindow(): IDescriptionText {
```